compose.yaml

```diff
@@ -3,19 +3,24 @@ networks:
   vault-pki-exporter:
 services:
   vault:
-    image: vault:1.13.3
+    image: hashicorp/vault:1.14.1
     networks:
       - vault-pki-exporter
     ports:
       - 8200:8200
     environment:
-      - VAULT_ADDR=http://vault:8200
+      - VAULT_ADDR=http://127.0.0.1:8200
```

Comment:
is it working ? because each container has different address and normally there are not in host network mode ?

Reply:
yes it does work! 127.0.0.1 is fine too but the docker dns name seems a bit more foolproof to me